cmd/cli/plugin/package/package_available_get.go

```diff
@@ -156,3 +169,50 @@ func getValuesSchemaForPackage(namespace, name, version string, kc kappclient.Cl
 
 	return nil
 }
+
+func generateDefaultValuesForPackage(pkg *kapppkg.Package, valuesFile ...*os.File) error {
+	if pkg == nil {
+		// should not happen
+		return errors.New("pkg is nil")
+	}
+
+	if len(pkg.Spec.ValuesSchema.OpenAPIv3.Raw) == 0 {
+		log.Warningf("package '%s/%s' does not have any user configurable values in the '%s' namespace", pkgName, pkgVersion, packageAvailableOp.Namespace)
+		return nil
+	}
+
+	fileNamePrefix := ""
+	pkgNameTokens := strings.Split(pkgName, ".")
+	if len(pkgNameTokens) >= 1 {
+		fileNamePrefix = fmt.Sprintf("%s-", pkgNameTokens[0])
```

Comment:
Nit pick: Is it better to just use `pkgNameTokens[0]` here and move the `-` to line 189?